portfolio/authkey.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "lijer-step-2020",
+  "private_key_id": "91ab91108db63dcaa209b5dd56e6b2f8cb327c31",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQDQTeydWjK4+gsu\nxRZ/RuJVUrxHGq4pKRkYlHIhB28dmuoPfPA7IR4DE79hPWrA/OUGcIgwDYrJlM0T\nQ+65tLpzgfKBO8Fxc0rJJQRZzylQNQ2AUXTp2nzJdIkDUCxtBZ3B3X8RkZL5RR4Z\nTC145XFdDW8LnUt/4mX6vUTY33pqLyF/QpWdkPxUycPAC6HiM3ZB6gAMT4QsbrKb\nyXcKsmJT4qta7o1VM8e2BQB1fNB/LncBFrsVVp+UiSEinSyAt83YWbA5Jt53+fcB\nNMPrl0nRnTELgRbzlXJi6B8r87vRCCuoY6rBi/L5ioez5eSlw9GxTIyf3QMi8h2s\nWsmWhRp/AgMBAAECggEAREt4K3L6NFmOtwvZP2uPI2hX4Hwn48jcuIlbDecxECTN\nXryYytoly956nbgUOyfvMvhMPm0CNus5fXqFV4+gneRaDPL5jme9ektPrN5V7j0m\nqw6J5W5EAYzaqyJrHmd7/KbRVYJEDMQThTnLotqfG2IfKVCuA0TIKLhLTUugCBXK\nP00Uo78epMDsShESdw5I7bXIu+kgiRmo0PWiHZetN9mu8iy6DINEjVm7sdvp0ObN\nG1G2xmN7cyzkHyEx9asqJNeAmP9Cx2Vh3e8yrWCpoaIFGBO6hGoh0pE0uOd/Izda\ncGsll++W4Vhe/IahqMfxUiPkSofoDaMUEydKQAyWcQKBgQDvrvInh9zAQkp03MgQ\nBJDcopQsVTzDxrJisC2TME3cD7Iw+XUSJAq5z3g0PiXmDuz61s7TSriBWiNdodPH\niCfqOjk074sKIpaH5Q99+JUzMMi4wBmERN47cxgwDEBjFVJuZ0G2ugVRvp+K02Lt\nwVCf8vksQAA7pPeM/lek+H9l2QKBgQDefB/qitjdubG0YoDKWXA6JGh9wgkz9lXv\nQ+Axco0YMmcVSmaSHLAi8hFOg7bOErLD09h7nxSF69rsmEcWhIdD97OwHvgz5rpC\n7aroReo2VxvyQkS9oaNZ0Cr3haK83Qo6EfIL0i8WKyoU02L/GKmkEinKdArthxFW\nKps/H04UFwKBgHsxi4uf/GnhaM26MSZygla1Rb2nrgzekw/AiBxQRqSmeKtNfMvv\nNIgeyCTqEKa34zxmrnXRwRXTU7wrkmPLauCsPn0j5m42KvL8XEMeQunF2FWFf1gJ\n4cPiu3ruwSVsbysJ7a/S2ysR9ITxc7bKlvsvcr04xCXj+80/aaMEis65AoGBANy0\nu6CbDpmeHbg5Q1P4t4kmFvYyJ5kBlKwfZGcJIQjQ70vCm8/0X5K7GgoORalWCTqU\njBvOariTiYDNay/eVqVJM1rSzhwGld3REYsL2FUBwvpnXPPVRkvRYt5bFl5O1FdY\n+Alb/1P5i+xO6tTurVLSpdxtk22gYMVezNUog/cPAoGBALLcWrpd9PFHXTZ0mH8y\nMsrk2PVS8eaGmPPOy7zNYvmoZ+NkVYcyXSz/6seMnDja0TQYoc/HmqGFJ8g3p+8F\nv4dE6vNICmrNCOrgD+2zvjtcguhPLqicEzS7YmtDoBc5W9k3rTdnwZllYIRk9yOw\n8N7+p60vsqOTYPlJamRzRkW4\n-----END PRIVATE KEY-----\n",
```

Comment:
Intern hosts got advice to not let you check the API keys into Github projects that are open to the public. Can you use a .gitignore file to store a .json file or .txt file with the API key? https://screenshot.googleplex.com/BrCgvRTGm7z

Reply:
got it! I just removed the key from the github repository because the website is now deployed and shouldn't need the key anymore.